src/views/CoCurricularTranscript/index.js

```diff
@@ -241,8 +244,21 @@ export default class Transcript extends Component {
     return minorsString.substr(0, minorsString.length - 2);
   };
 
+  userIsFacStaff = () => {
+    return this.state.profile.PersonType?.includes('fac');
+  };
+
   render() {
-    if (this.props.authentication) {
+    if (this.userIsFacStaff()) {
```

Comment:
Since it's only called once now, you could remove the function and just put the boolean here directly.